### ISSUE TITLE
HTTP error 500

### ISSUE THREAD

**Issue by inghoward** [erlymon/erlymon-litvak-monitor-android#1]
Hello! Always when I try to send commands to devices got an Error 500. Do you know something about this error? Thanks for your help

**Comment by erlymon** [comment]
Can you see that the server responds when you execute the command?

**Comment by inghoward** [comment]
Yes, server is up and running without issues and can be normally accessed on-line.

**Comment by rafael31** [comment]
it happens to me too. My server its up and i can send command form a browser.

When i send the command from the app, the log on my server doesn't show anything. Only the app show http 500 server error

**Comment by pese-git** [comment]
Can you give me a test access to an your server? If possible, please send me the necessary data by e-mail support@erlymon.org

**Comment by erlymon** [comment]
Can you give me a test access to your server? If possible, please send me the necessary data by e-mail support@erlymon.org

**Comment by erlymon** [comment]
I fixed a bug with the command. After a while you will be able to download the update from Google Play.
https://play.google.com/apps/testing/org.erlymon.litvak.monitor

**Comment by inghoward** [comment]
Thanks Erlymon! I Will be waiting for the play update.

**Comment by inghoward** [comment]
Well, in my device the application die every time I send a message. I tested using a tk103 GPS. 
I tried cleaning cache and default data. Nothing change, still dies.

**Comment by erlymon** [comment]
Perhaps Google produces a report of your bug, then I can fix it.

**Comment by inghoward** [comment]
Hello!
Have Google produced the bug report?

**Comment by erlymon** [comment]
I do not recived bug reports.
